app/assets/stylesheets/base/_fonts.scss

```diff
@@ -5,22 +5,22 @@
 
 @font-face {
   font-family: 'bpicons';
-  src:url('bpicons.eot');
-  src:url('bpicons.eot?#iefix') format('embedded-opentype'),
-    url('bpicons.woff') format('woff'),
-    url('bpicons.ttf') format('truetype'),
-    url('bpicons.svg#bpicons') format('svg');
+  src:font-url('bpicons.eot');
+  src:font-url('bpicons.eot?#iefix') format('embedded-opentype'),
+    font-url('bpicons.woff') format('woff'),
+    font-url('bpicons.ttf') format('truetype'),
+    font-url('bpicons.svg#bpicons') format('svg');
   font-weight: normal;
   font-style: normal;
 } /* Made with http://icomoon.io/ */
 
 @font-face {
   font-family: 'fontawesome';
-  src:url('fontawesome.eot?-e43dk9');
-  src:url('fontawesome.eot?#iefix-e43dk9') format('embedded-opentype'),
-    url('fontawesome.woff?-e43dk9') format('woff'),
-    url('fontawesome.ttf?-e43dk9') format('truetype'),
-    url('fontawesome.svg?-e43dk9#fontawesome') format('svg');
+  src:font-url('fontawesome.eot?-e43dk9');
+  src:font-url('fontawesome.eot?#iefix-e43dk9') format('embedded-opentype'),
```

Comment:
Colon after property should be followed by one space<br>Prefer double-quoted strings